policy-controller/src/main.rs

```diff
@@ -232,3 +295,33 @@ impl std::str::FromStr for LogFormat {
         }
     }
 }
+
+// Load public certificate from file.
+fn load_certs(filename: &str) -> Result<Vec<rustls::Certificate>, String> {
+    // Open certificate file.
+    let certfile =
+        fs::File::open(filename).map_err(|e| format!("failed to open {}: {}", filename, e))?;
+    let mut reader = io::BufReader::new(certfile);
+
+    // Load and return certificate.
+    let certs = rustls_pemfile::certs(&mut reader)
+        .map_err(|e| format!("failed to load certificate: {}", e))?;
+    Ok(certs.into_iter().map(rustls::Certificate).collect())
+}
```

Comment:
a nicer solution, rather than turning all the errors into strings here, might be to use `anyhow`'s [`with_context`](https://docs.rs/anyhow/latest/anyhow/trait.Context.html#tymethod.with_context) to add the context that we were loading the certificate, at the callsite. that reduces the error-handling boilerplate here:
```suggestion
// Load public certificate from file.
fn load_certs(filename: &str) -> anyhow::Result<Vec<rustls::Certificate>> {
    // Open certificate file.
    let certfile = fs::File::open(filename)?;
    let mut reader = io::BufReader::new(certfile);

    // Load and return certificate.
    let certs = rustls_pemfile::certs(&mut reader)?;
    Ok(certs.into_iter().map(rustls::Certificate).collect())
}
```

then, when we call `load_certs`, we could write something like this:
```rust
                 let certs = match load_certs("/var/run/linkerd/tls/tls.crt")
                      .with_context(|| "failed to load certificate") {
                    Ok(certs) => certs,
                    Err(error) => {
                        error!(%error);
                        continue;
                     }
                 };
```